wordpress-mobile/WordPress-Android
# Domains Dashboard

builds domains list screen based on i2.


Fixes #15298 

<table><tr>
<td><img width=320 src="https://user-images.githubusercontent.com/990349/134456350-38b5fc1e-fa9b-4d1d-9ae2-10977459e407.png" /></td>
<td><img width=320 src="https://user-images.githubusercontent.com/990349/134456381-33c48faf-2291-42ec-9650-b94cbcdceaac.png"</td>
</tr></table>


To test:

1. Enable AppSettings -> Debug settings -> SiteDomainsFeatureConfig ✅
2. Return to **My Site**, Notice there's a new item under **Configuration** -> 🌐 Domains
3. Click on Domains, takes your Site Domains Screen
4. Test with a free plan and paid plan to to see the above screens

NOTE: It is work in progress, design and behavior will be improved in the next iterations.  
All links point to existing register domain flow 
The kebab menu, manage domains and learn more all will be implemented separately. 
The copy and design will be updated as it evolves in the next PRs

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
